entity/contents.cc

```diff
@@ -235,8 +235,7 @@ bool TextureContents::Render(const ContentContext& renderer,
               auto coverage_coords =
                   (vtx - coverage_rect->origin) / coverage_rect->size;
               data.texture_coords =
-                  (source_rect_.origin +
-                   source_rect_.size * coverage_coords) /
+                  (source_rect_.origin + source_rect_.size * coverage_coords) /
```

Comment:
How did this change?

Reply:
Formatting

Reply:
Not sure why...  Different version of clang?

Reply:
Possibly. Not sure. Don't care much TBH.